### ISSUE TITLE
German Translation updated for v0.6.12b2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#323]
```
New strings in v0.6.12b2 have been translated:
 > content_itemcategories.xml
 > strings.xml

Small improvements to the files:
 > content_itemlist.xml
 > content_monsterlist.xml
 > content_questlist.xml

Translated conversations have been reviewed until end of Fallheaven:
 > content_conversationlist.xml

Further review needed for Vilegard.

Phoenixor and Tutar need to be added as Authors.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 24 Oct 2012 at 12:40

Attachments:
- [language update.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-323/comment-0/language update.zip)

**Comment by GoogleCodeExporter** [comment]
```
There are also 2 lines fixed in the English "content_conversationlist.xml".
```

Original comment by `SamuelPl...@gmail.com` on 24 Oct 2012 at 12:40

**Comment by GoogleCodeExporter** [comment]
```
Looks good, great work Samuel.
Merged in 9c95e97a01b67165f8b1296cf55f671fd7701a80.
```

Original comment by `oskar.wi...@gmail.com` on 26 Oct 2012 at 7:09
- Changed state: **Ready**
- Added labels: **Milestone-0.6.12**, **Type-Translation**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:44
- Changed state: **Fixed**
